TramsDataApi/Setup/HealthCheckSetup.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
Same as above, you could make it to pass in the context and name, as use the same method (reducing code duplication)